mwdb/web/src/App.js

```diff
@@ -36,12 +36,14 @@ import UserPasswordRecover from './components/UserPasswordRecover';
 import { library } from '@fortawesome/fontawesome-svg-core'
 import { faTimes, faUpload, faDownload, faPlus, faMinus, faRandom, 
          faExchangeAlt, faBan, faSearch, faToggleOn, faToggleOff, faSort, faSortUp, faSortDown, faProjectDiagram, faFile, faFileImage, faFilePdf,
-         faFingerprint, faBoxes, faTrash, faCopy } from '@fortawesome/free-solid-svg-icons'
+         faFingerprint, faBoxes, faTrash, faCopy, faThumbtack, faStar} from '@fortawesome/free-solid-svg-icons'
```

Comment:
Missing space (formatting)
```suggestion
         faFingerprint, faBoxes, faTrash, faCopy, faThumbtack, faStar } from '@fortawesome/free-solid-svg-icons'
```